diff -Nur Newtonsoft.Json-8.0.3.orig/Makefile Newtonsoft.Json-8.0.3/Makefile
--- Newtonsoft.Json-8.0.3.orig/Makefile	1970-01-01 01:00:00.000000000 +0100
+++ Newtonsoft.Json-8.0.3/Makefile	2016-06-15 15:20:54.654705936 +0200
@@ -0,0 +1,10 @@
+all:
+	cd Src/Newtonsoft.Json ; xbuild Newtonsoft.Json.Net40.csproj /property:SignAssembly=true /property:AssemblyOriginatorKeyFile=Dynamic.snk /property:Configuration=Release /property:DefineConstants='SIGNED NET40'
+
+install:
+	install -d -m 755 /usr/lib
+	install -m 644 Src/Newtonsoft.Json/bin/Release/Net40/Newtonsoft.Json.dll /usr/lib
+	install -m 644 Src/Newtonsoft.Json/bin/Release/Net40/Newtonsoft.Json.dll.mdb /usr/lib
+	install -d -m 755 /usr/share/pkgconfig/
+	install -m 644 Newtonsoft.Json.pc /usr/share/pkgconfig/
+
diff -Nur Newtonsoft.Json-8.0.3.orig/Newtonsoft.Json.pc Newtonsoft.Json-8.0.3/Newtonsoft.Json.pc
--- Newtonsoft.Json-8.0.3.orig/Newtonsoft.Json.pc	1970-01-01 01:00:00.000000000 +0100
+++ Newtonsoft.Json-8.0.3/Newtonsoft.Json.pc	2016-06-15 15:08:07.847270907 +0200
@@ -0,0 +1,11 @@
+prefix=/usr
+exec_prefix=/usr
+libdir=/usr/lib
+
+Name: Newtonsoft.Json
+Description: Json.NET is a popular high-performance JSON framework for .NET
+Requires:
+Version: 8.0.0.0
+Libs: -r:${libdir}/Newtonsoft.Json.dll
+Cflags:
+
